src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard/MovieCard';
+
+import { Movies } from '../../types/Movies';
+
+export const MoviesList: React.FC<Movies> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard {...movie} />
```

Comment:
Let's not use spread operator here

Reply:
And also, always use `key` when mapping components

Reply:
> 

is spread that bad?)

Reply:
It's not bad, but you might pass unnecessary properties, like imdbId which you don't use at all